upgrade_testing/data/upgrade

```diff
@@ -291,14 +291,21 @@ function do_normal_upgrade() {
     # Allow upgrade from lts to non-lts if there's not lts to upgrade to
     local version
     local dev_version
-    if grep '^Prompt=lts' /etc/update-manager/release-upgrades; then
-        # Check for an LTS to LTS upgrade
-        version=$(do-release-upgrade -c | awk '/New release/ {print $3}' | tr -d \')
-        dev_version=$(do-release-upgrade -d -c | awk '/New release/ {print $3}' | tr -d \')
-        if [ -z "${version}" ] && [ -z "${dev_version}" ]; then
-            upgrade_log "No LTS version available, allowing 'normal' upgrades"
-            # No LTS release to upgrade to. Enable non-LTS upgrades.
-            sed -i 's/Prompt=lts/Prompt=normal/' /etc/update-manager/release-upgrades
+
+    if [ "$PROMPT" != "" ]; then
```

Comment:
Nit: this is normally done using `[ -n "$PROMPT" ]` (similar to `[ -z "${version}" ]` a few lines below), but there is no practical difference between the two AFAIK.

Reply:
I like to write it like that, because it's easier for someone not very familiar to `bash` to figure out the intent without having to `man test` and check exactly what that `-n` flag tests. :D